interfaces/mocha.js

```diff
@@ -0,0 +1,4 @@
+declare function describe(message: string, impl: Function): void;
+declare function xdescribe(message: string, impl: Function): void;
+declare function it(message: string, impl: Function): void;
+declare function xit(message: string, impl: Function): void;
```

Comment:
That's very strange; adding these libs definitely resolved a bunch of flow errors.